doc/source/whatsnew/v1.5.0.rst

```diff
@@ -79,6 +79,7 @@ as seen in the following example.
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
+- :meth:`Series.map` now raises when ``arg`` is dict but ``na_action`` is not either None or 'ignore' (:issue:`46588`)
```

Comment:
can you use double back-ticks around ``None`` and ``'ignore'``

Reply:
Yeah definitely! Done.